acceptance-tests/src/test/java/gov/di_ipv_drivingpermit/pages/DrivingLicencePageObject.java

```diff
@@ -708,6 +718,15 @@ public void assertInvalidIssueDateInErrorSummary(String expectedText) {
         Assert.assertEquals(expectedText, InvalidIssueDateErrorInSummary.getText());
     }
 
+    public void assertInvalidIssueDateInErrorSummaryDVA(String expectedText) {
```

Comment:
This is already in DVAEnterYourDetailsExactlyPageObject as assertInvalidIssueDateErrorText can we please keep DVA and DVLA separate and not duplicate them